src/toolbar.scss

```diff
@@ -141,67 +155,63 @@ $title-toolbar-height: 2.75rem;
     background-color: var(--sapToolbar_SeparatorColor);
   }
 
-  &__overflow {
-    &.#{$fd-namespace}-popover {
-      .#{$block}__overflow__body {
-        @include fd-reset();
+  &__body {
+    &--overflow {
+      @include fd-reset();
+      @include fd-flex(column);
 
-        display: flex;
-        flex-direction: column;
-        max-width: 20rem;
-        padding: 0.1875rem 0.375rem;
+      max-width: 20rem;
+      padding: 0.1875rem 0.375rem;
+      max-height: 50vh;
+      overflow: auto;
 
-        > * {
-          display: block;
-          text-align: left;
-          width: auto;
-          margin: 0.1875rem 0;
+      & .#{$block}__button {
+        display: block;
+        text-align: left;
+        width: auto;
+        margin: 0.1875rem 0;
 
-          @include fd-rtl() {
-            text-align: right;
-          }
+        @include fd-rtl() {
+          text-align: right;
         }
 
-        > .#{$block}__separator {
-          height: 0.0625rem;
-          margin: 0.1875rem;
-          width: auto;
-          background-color: var(--sapToolbar_SeparatorColor);
-        }
-
-        .#{$block}__overflow__form-label {
-          &.#{$fd-namespace}-form-label {
-            margin-top: 0.4375rem;
+        &--menu {
```

Comment:
Where's the example in the documentation?

Reply:
Added menu button in documentation.